.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+registries:
+  rubygems-server-enterprise-contribsys-com:
+    type: rubygems-server
+    url: https://enterprise.contribsys.com/
+    username: "${{secrets.RUBYGEMS_SERVER_ENTERPRISE_CONTRIBSYS_COM_USERNAME}}"
+    password: "${{secrets.RUBYGEMS_SERVER_ENTERPRISE_CONTRIBSYS_COM_PASSWORD}}"
```

Comment:
These secrets will need to be set. Currently the user:pass is set in a single variable here: `BUNDLE_ENTERPRISE__CONTRIBSYS__COM`. I think we should use one or the other. Perhaps we should update our GHA PR to combine the two.

Also, the name seems long. Is there a reason for it to be this long?

Reply:
This was just the default that dependabot created